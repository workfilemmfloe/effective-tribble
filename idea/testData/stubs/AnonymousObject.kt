class A
trait T
val obj = object : A(), T