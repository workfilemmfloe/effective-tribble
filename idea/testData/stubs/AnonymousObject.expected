PsiJetFileStubImpl[package=]
  PACKAGE_DIRECTIVE:
  IMPORT_LIST:
  CLASS:[fqName=A, isEnumEntry=false, isInterface=false, isLocal=false, isTopLevel=true, name=A, superNames=[]]
  CLASS:[fqName=T, isEnumEntry=false, isInterface=true, isLocal=false, isTopLevel=true, name=T, superNames=[]]
  PROPERTY:[fqName=obj, hasDelegate=false, hasDelegateExpression=false, hasInitializer=true, hasReturnTypeRef=false, isExtension=false, isTopLevel=true, isVar=false, name=obj]
    OBJECT_DECLARATION:[fqName=null, isCompanion=false, isLocal=true, isObjectLiteral=true, isTopLevel=false, name=null, superNames=[A, T]]
      DELEGATION_SPECIFIER_LIST:
        DELEGATOR_SUPER_CALL:
          CONSTRUCTOR_CALLEE:
            TYPE_REFERENCE:
              USER_TYPE:[isAbsoluteInRootPackage=false]
                REFERENCE_EXPRESSION:[referencedName=A]
        DELEGATOR_SUPER_CLASS:
          TYPE_REFERENCE:
            USER_TYPE:[isAbsoluteInRootPackage=false]
              REFERENCE_EXPRESSION:[referencedName=T]
      CLASS_BODY:
